css/index.less

```diff
@@ -0,0 +1,160 @@
+/* colors used throughout the page */
+@dark-brown: hsl(31, 49%, 10%);
+@light-brown: lighten(@dark-brown, 50%);
+@near-black: hsl(220, 6%, 10%);
+@red: hsl(0, 100%, 50%);
+@lighter-brown: lighten(@light-brown, 15%);
+/* set box-sizing, margin, padding, and color for html and body */
+html, body {
+  -webkit-box-sizing: border-box;
+  -moz-box-sizing: border-box;
+  box-sizing: border-box;
+  margin:0;
+  padding:0;
+  color: @dark-brown;
+}
+/* set background image for the body */
+body {
+  background-image: url('../img/shopping-center-iot.jpg');
+  -webkit-background-size: cover;
+  -moz-background-size: cover;
+  -o-background-size: cover;
+  background-size: cover;
+  height: 45.938rem;
+  background-position: center bottom;
+  overflow: auto;
+  padding:0;
+  margin: auto;
+  font-family: 'Architects Daughter', cursive;
+  background-repeat: no-repeat;
+}
+/* add fornt to h1 and h3 */
+h1, h3 {
+  font-family: 'Bangers', cursive;
+}
+/* set font size for h1 */
+h1 {
+  font-size: 64px;
+}
+/* reset components */
+*, *:before, *:after {
+  -webkit-box-sizing: inherit;
+  -moz-box-sizing: inherit;
+  box-sizing: inherit;
+}
+/* set color, radius, border and opacity of class */
+.content-background {
+  background-color: @lighter-brown;
+  border-bottom-right-radius: 25px;
+  border-top-left-radius: 25px;
+  border: 1px solid black;
+  opacity: 0.9;
+  filter: alpha(opacity=60);
+}
+/* set margin and padding for content */
+.content {
+  margin: 0 auto;
+  padding: 1rem;
+  /* set padding for content ul */
+  ul {
+    padding: 0;
+  }
+  /* set margin and display for content li */
+  li {
+    display: block;
+    margin-bottom: 5px;
+  }
+  /* set border, width, height, shadow, and padding for content i */
+  i {
+    border-radius: 1.25rem;
+    width: 1rem;
+    height: 1rem;
+    box-shadow: 0px 0px 3px @near-black;
+    padding-left: .1rem;
+  }
+}
+/* set float for listnames and listitems */
+.listNames, .listItems {
+  float: left;
+  /* set border color for listnames and listitems error */
+  .error {
+    border-color: @red;
+  }
+}
+/* align text center for title, listitemstitle, and form */
+.title, .listItemsTitle, form {
```

Comment:
Consider refactoring this into a single class you can add to anything you need centered. Parsing 1 class is faster than comma lists like this